lib/generators/hyrax/work/templates/controller.rb.erb

```diff
@@ -1,7 +1,7 @@
 # Generated via
 #  `rails generate hyrax:work <%= class_name %>`
-
 module Hyrax
+  # Gererated controller for <$= class_name %>
```

Comment:
There are two typos here: Please change `Gererated` to `Generated` and `<$` to `<%` in a follow-on PR, @aaron-collier!

Reply:
Sorry, I missed those!